tools/targetbrowser/client.go

```diff
@@ -0,0 +1,59 @@
+package targetbrowser
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"time"
+
+	"github.com/pkg/errors"
+)
+
+type Client struct {
+	apiKey     string
+	baseURL    string
+	HTTPClient *http.Client
+}
+
+// NewClient creates new trilio backend client with given API key
+func NewClient(apiKey string) *Client {
+	return &Client{
+		apiKey: apiKey,
+		HTTPClient: &http.Client{
+			Timeout: 5 * time.Minute,
+		},
+		baseURL: "http://pankaj1.k8s-tvk.com/sample-target.pankaj1/",
```

Comment:
remove this.. and all other hardcoded changes which you made for local testing

Reply:
done